src/mongo/MongoScrapbook.ts

```diff
@@ -201,7 +201,7 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 	}
 
 	visitFunctionCall(ctx: mongoParser.FunctionCallContext): MongoCommand[] {
-		if (ctx.parent instanceof mongoParser.CommandContext) {
+		if (ctx.parent instanceof mongoParser.CommandContext && ctx._FUNCTION_NAME) {
 			this.commands[this.commands.length - 1].name = (ctx._FUNCTION_NAME && ctx._FUNCTION_NAME.text) || "";
```

Comment:
What does `(ctx._FUNCTION_NAME && ctx._FUNCTION_NAME.text)` resolve to anyway?  Are those 2 strings?

Reply:
`ctx._FUNCTION_NAME` is a Token that the parser identifies. 
`ctx._FUNCTION_NAME.text` is the string the token points to. 

Should resolve to a string, else a false-y value.